test/basics.m

```diff
@@ -296,6 +297,15 @@ - (void)secret {}
   EXPECT_NO_MESSAGES ();
 }
 
+TESTCASE (module_name_can_be_retrieved)
+{
+  COMPILE_AND_LOAD_SCRIPT (
+      "var NSString = ObjC.classes.NSString;"
+      "var badger = NSString.stringWithString_(\"badger\");"
+      "send(badger.$moduleName);");
+  EXPECT_SEND_MESSAGE_WITH ("\"/System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation\"");
```

Comment:
```suggestion
  EXPECT_SEND_MESSAGE_WITH ("\"/System/Library/Frameworks/"
      "CoreFoundation.framework/Versions/A/CoreFoundation\"");
```

Reply:
Interestingly the path changes between `NSString.$moduleName` and the `badger` -- updated test incoming w/ de39cae